src/ecModules.ml

```diff
@@ -13,19 +13,16 @@ include EcCoreModules
 module OI = PreOI
 
 (* -------------------------------------------------------------------- *)
-let mr_empty = {
-  mr_xpaths = ur_empty EcPath.Sx.empty;
-  mr_mpaths = ur_empty EcPath.Sm.empty;
-}
-
-let mr_full = {
-  mr_xpaths = ur_full EcPath.Sx.empty;
-  mr_mpaths = ur_full EcPath.Sm.empty;
-}
-
-let mr_add_restr mr (rx : Sx.t use_restr) (rm : Sm.t use_restr) =
-  { mr_xpaths = ur_union Sx.union Sx.inter mr.mr_xpaths rx;
-    mr_mpaths = ur_union Sm.union Sm.inter mr.mr_mpaths rm; }
+(* Noting is restricted. *)
```

Comment:
Nothing

Reply:
Done, + the original one since it was a cut and past.